.github/actions/trigger-github-release/entrypoint.sh

```diff
@@ -5,21 +5,18 @@ set -e
 echo "INFO: Installing action dependencies..."
 (cd /action; npm ci)
 
-echo "INFO: Checking out current commit..."
-git -c advice.detachedHead=false checkout $GITHUB_SHA
-
 # GitHub doesn't support running actions on new tags yet: we need to run it on the commit.
 # For this reason, we can't be sure that the tag already exists. We can use the commit
 # message to create the tag. If the tag already exists locally, they won't conflict because
 # they have the same name and are on the same commit.
 echo "INFO: Getting release version..."
-# current_tag=$(git describe --abbrev=0 --tags HEAD)
-current_tag=$(git log --oneline --format=%B -1 HEAD)
+# current_tag=$(git describe --abbrev=0 --tags $GITHUB_SHA)
```

Comment:
could you remove this line since its commented?

Reply:
The commented line is the "correct" way to get the last release, but it will only work when this action is fired when the new tag is pushed. Currently GitHub only supports this for private repos.